product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -149,6 +151,17 @@ When enabling read-only workloads, keep in mind the following:
     
 For information on replication lag while using read-only workloads, see [Synchronous replication](/biganimal/latest/overview/02_high_availability/#synchronous-replication). 
 
+#### PgBouncer
```

Comment:
Consider reordering the info a bit.

- Describe the benefits of PgBouncer. 
- Describe the two deployment options mentioning the server side deployment incurs additional cost.
- Then how to enable the server side deployment. 
You have all the right words (with one minor exception). I think we just need to put them together a little differently.